contracts/multihop/src/contract.rs

```diff
@@ -67,7 +67,7 @@ impl MultihopTrait for Multihop {
                 &op.offer_asset,
                 &next_offer_amount,
                 &None::<i64>,
-                &Some(5000i64),
+                &None::<i64>,
```

Comment:
Wait, where does this change comes from? If anything we should add this parameter to this message, not enforce `None`. And it should be in a separate PR.

Reply:
thought it won't hurt us, reverting now